ext/src/ruby_api/externals.rs

```diff
@@ -25,7 +25,9 @@ impl DataTypeFunctions for Extern {
 
 impl Extern {
     /// @yard
-    /// @return [Wasmtime::Func] The exported function, if this is a function.
```

Comment:
Nit: we refer to Wasmtime classes elsewhere without the namespace, updated for consistency.